build.gradle

```diff
@@ -63,6 +64,12 @@ allprojects {
     targetCompatibility = JavaVersion.VERSION_1_8
   }
 
+  tasks.withType(KotlinCompile) {
```

Comment:
we should probably add a comment here indicating that this can be removed when we drop 2020

Reply:
Or rather, upped to 1.4, bundled with 2020.3? https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library

I'll add a comment.